manuscripts/esquery.py

```diff
@@ -241,9 +241,11 @@ def __get_bounds(cls, start=None, end=None):
             start_ts = None
             end_ts = None
             if start:
+                start = start.replace(microsecond=0)
                 start_ts = start.replace(tzinfo=timezone.utc).timestamp()
                 start_ts_ms = start_ts * 1000  # ES uses ms
             if end:
+                end = end.replace(microsecond=0)
```

Comment:
Why do we need to set microsecond to 0? Is not configured in this way by default?

Reply:
@acs: No, it isn't configured this way by default and results in an error from elasticsearch.
![screen shot 2018-06-04 at 11 46 57 am](https://user-images.githubusercontent.com/9946566/40901253-0775277c-67ed-11e8-86d9-2a8c721a6ebb.png)
This error presents it self when we convert datetime to timestamp.
I'll update the file and mention it there.

Reply:
Ah, ok, it is related with the migration to elasticsearch_dsl. Great! I want just to understand the need for it. Thanks you @aswanipranjal ! LGTM